singer-commons/src/main/java/com/pinterest/singer/loggingaudit/client/AuditEventKafkaSender.java

```diff
@@ -165,6 +165,13 @@
 
   private Map<LoggingAuditHeaders, Integer> eventTriedCount = new ConcurrentHashMap<>();
 
+  /**
+   *  Default partition of the audit_event topic used to receive audit events. Default partition
+   *  will be reset in resetDefaultPartitionIfNeeded() method. This reduces the number of TCP
+   *  connections from audit client to the Kafka Cluster hosting the audit_event topic.
+   */
+  private int defaultPartition = -1;
```

Comment:
Should this be called `currentPartitionId` instead of defaultPartition?

Reply:
sure. currentPartitionId sounds better.  will update this